depenguinme.sh

```diff
@@ -119,6 +119,7 @@ done
 apt-get update
 apt-get install -y mkisofs
 apt-get install -y qemu
+apt-get install -y qemu-system-x86
```

Comment:
What about changing all of these based on if the binary could be found or not?

something like

```sh
if ! command -v thecommand &> /dev/null
then
    apt-get install -y thepackage
fi
```

Reply:
might be useful in future, especially if checking which platform script is running on. for now apt will exit if already installed

Reply:
I can add that myself in a separate review in a generic way. This would allow to skip all apt steps (including update) if nothing is missing.